src/components/tou-modal/tou-banner.jsx

```diff
@@ -0,0 +1,44 @@
+import React, {useState} from 'react';
+import {injectIntl, FormattedMessage} from 'react-intl';
+import TosModalReminderUnder16 from './variations/tou-reminder-under-16.jsx';
+
+require('./tou-modal.scss');
+
+const TermsOfUseLink = chunks => {
+    const [isModalVisible, setIsModalVisible] = useState(false);
+    return (<>
+        <a
+            className="tou-banner-link"
+            // eslint-disable-next-line react/jsx-no-bind
+            onClick={() => {
+                setIsModalVisible(true);
+            }}
+        >
+            {chunks}
+        </a>
+        {isModalVisible && <TosModalReminderUnder16
+            // eslint-disable-next-line react/jsx-no-bind
+            onClose={() => {
+                setIsModalVisible(false);
+            }}
+            isOpen={isModalVisible}
+        /> }
+    </>);
+
+};
+
+// eslint-disable-next-line arrow-body-style
```

Comment:
We don't have to disable the rule but omit the return. And do it like this:
`const TermsOfUseBanner = () => (
    <div className="tou-banner">
        <FormattedMessage
            id="tos.banner.parentAgreement"
            values={{
                a: TermsOfUseLink
            }}
        />
    </div>
);`
Same for the other similar components